doc/conf.py

```diff
@@ -149,9 +151,15 @@
     html_baseurl = "https://pygmt.org/latest/"
 release = __version__
 
+minimum_python_version = metadata("pygmt")["Requires-Python"]
+with pygmt.clib.Session() as lib:
+    minimum_gmt_version = lib.required_version
```

Comment:
Maybe we should define a package level variable `__minimum_gmt_version__` to simplify the codes and also the release checklist for bumping the minimum required GMT version (https://github.com/GenericMappingTools/pygmt/blob/main/.github/ISSUE_TEMPLATE/bump_gmt_checklist.md)?